app/controllers/metadata_controller.rb

```diff
@@ -51,7 +51,7 @@ def default_manifest
       icons: [
       ],
       share_target: {
-        action: "/new-topic",
+        action: "#{Discourse.base_uri}/new-topic",
```

Comment:
I'm not sure this should be a URL

Reply:
Yeah I thought the same, since the key is `action`, not `url`. However:

https://w3c.github.io/web-share-target/

> When a share takes place, if the user selects this share target, the user agent opens a new browsing context at the action URL, with query parameter values containing the shared data, just like an HTML form submission.

Reply:
Hmm, but the example [here](https://web.dev/web-share-target/) is a path:

```json
"share_target": {
  "action": "/share-target/",
  "method": "GET",
  "params": {
    "title": "title",
    "text": "text",
    "url": "url"
  }
}
```

Reply:
Oh I think I understand the question now. `Discourse.base_uri` is not a full URL. For most sites it is an empty string. For subfolders, it will be something like `/forum`

So adding this will not result in a URL, it will remain a path.

Reply:
Ohhh, I should have known. Looks good to me then :+1: